.github/workflows/publish-helm-chart.yaml

```diff
@@ -4,6 +4,8 @@
 name: Publish Helm chart
 
 on:
+  tags:
+    - "v*"
```

Comment:
So far it was manually dispatched workflow. If we want to publish on each release, it should probably a step executed in the Release workflow after we publish docker images. WDYT?